guides/v9-hammerjs-migration.md

```diff
@@ -128,15 +128,17 @@ plugin, or to an actual `@Output`. For example:
 ```
 
 In the example above, `rotate` could be an event from the deprecated `GestureConfig`, or an
-`@Output` from `<image-rotator>`. The migration warns about this to raise awareness that it
-might have _incorrectly kept_ HammerJS. Please manually check if you can remove HammerJS.
+`@Output` from `<image-rotator>`. The migration warns you about this to raise awareness that it
+might have _incorrectly kept_ HammerJS. Please check if you can remove HammerJS from the project
+manually.
 
 **Case 2**: The deprecated Angular Material `GestureConfig` is used in combination with a
 custom [`HAMMER_GESTURE_CONFIG`][1]. This case is ambiguous because the migration is unable
 to detect whether a given HammerJS event binding corresponds to the custom gesture config, or to
-the deprecated Angular Material gesture config. If such a warning has been reported to you, check
-if you can remove references to the deprecated `GestureConfig`, or if you need to handle the events
-provided by the deprecated gesture config in your existing custom gesture config.
+the deprecated Angular Material gesture config. If such a warning has been reported, check
+if you can remove the references to the deprecated `GestureConfig`, or if you need to update your
+existing, custom gesture config to handle the events provided by the deprecated Angular Material
```

Comment:
The comma here seems wrong, right?

Reply:
I don't see an issue with any of the commas here?

The one on this line is separating 2 adjectives (existing and custom) where "gesture config" is the noun.

Reply:
I'm obviously not native, or an expert 😁 but for me it read like `existing` referred to something else, but not to the `custom gesture config`. I just checked, and I think I learned something new 😄 sorry for the noise!